core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -882,6 +882,9 @@
 
   @BaseMessage("Not a valid input for JSON_KEYS: ''{0}''")
   ExInst<CalciteException> invalidInputForJsonKeys(String value);
+
+  @BaseMessage("Not a valid input for JSON_REMOVE: jsonDoc: ''{0}'', kvs: ''{1}''")
```

Comment:
This is good but I am afraid that the camel case is still not very friendly to end users.